### DISCUSSION TITLE
Found duplicate topics on MM2 replication.

### DISCUSSION THREAD

**Discussion by vperi1730** [strimzi/strimzi-kafka-operator#4139] [Q&A]
Hi Scholzj,

I was testing an MM2 use case and found one behavior that I am not sure about. Following is the use case.

1) I have 2 clusters c202 and c204.
2) Both my clusters have topic created well before(tdec21 is the name)
3) c204 has MM2 up and running.
4) As MM2 does the auto replication I could see the same topic (tdec21) has been re-imported into the c204 though the topic has existed.

Question is, will MM2 not skip the auto replication as the topic already exists?. The output is something like this where 1st one is after replication, 2nd one already existed before.

```
my-source-cluster.tdec21                                        my-cluster205   3            3                    True
tdec21                                                          my-cluster205   3            1                    True
```

**Comment by scholzj** [comment]
TBH, I'm not sure I understand your setup and the problem. You would probably need to provide more details. When mirroring two Kafka clusters as active active, it is quite common that both of them contain a topic named `my-topic`. This topic is then mirrored to the other cluster with prefix. 

E.g. on Cluster A, you should see:
* its original `my-topic` 
* and the `cluster-b.my-topic` mirrored from cluster B.

Whereas on Cluster B you would see:
* its original `my-topic`
* and the `cluster-a.my-topic` mirrored form Cluster A

**Comment by vperi1730** [comment]
OK, Let's say if my target cluster already has a topic with a prefix(src-cluster.tdec21) that I created manually, will it re-mirror the topic from the src cluster again?

**Reply by scholzj** [reply]
I don't know - I would assume it would not, but I don't know for sure. But if it would, it should be prefixed once more.

**Comment by vperi1730** [comment]
OK, let me test this and update you back. Thank you :)
